Jenkinsfile

```diff
@@ -1,22 +1,35 @@
 pipeline {
-    agent { label 'pullrequest' }
+    agent { label 'aida-statedb' }
     
-    options { timestamps () }
+    options { 
+        timestamps ()
+        timeout(time: 1, unit: 'HOURS')
+    }
     
-    environment { 
-        PATH = '/usr/local/bin:/usr/bin:/bin:/usr/local/go/bin'
+    environment {
+        GOROOT = '/usr/lib/go-1.21/'
         STORAGE = '--db-impl carmen --db-variant go-file --carmen-schema 3'
         PRIME = '--update-buffer-size 4000'
         VM = '--vm-impl lfvm'
-        AIDADB = '--aida-db=/var/opera/Aida/mainnet-data/aida-db'
-        TMPDB = '--db-tmp=/var/opera/Aida/dbtmpjenkins'
-        DBSRC = '--db-src=/var/opera/Aida/dbtmpjenkins/state_db_carmen_go-file_${TOBLOCK}'
+        AIDADB = '--aida-db=/mnt/aida-db-central/aida-db'
+        TMPDB = '--db-tmp=/mnt/aida-statedb-disk'
+        DBSRC = '--db-src=${TMPDB}/state_db_carmen_go-file_${TOBLOCK}'
```

Comment:
The PR fails on Jenkins because DBSRC variable will resolve in `--db-src=--db-tmp=/mnt/aida-statedb-disk/state_db_carmen_go-file_4600000`